TAF_EasyRest/TestFramework/PageComponents/Client/WaitingForConfirmOrderPageComponent.cs

```diff
@@ -4,20 +4,21 @@ public class WaitingForConfirmOrderPageComponent
     {
         private IWebDriver driver;
         private int index;
+        private string number;
 
         public WaitingForConfirmOrderPageComponent(IWebDriver driver, int index)
         {
             this.driver = driver;
             this.index = index;
+            number = _orderField.FindElement(By.XPath("//div[@class='MuiGrid-item-120 MuiGrid-grid-xs-1-148'][1]")).Text;
```

Comment:
This is a bad idea to put such logic in the constructor because you can use the constructor before the actual page is displayed in the browser. I think you can make `number` as property (locator).